concent_api/conductor/tests/test_integration_verification.py

```diff
@@ -0,0 +1,290 @@
+from django.test    import override_settings
+from django.test    import TestCase
+from django.urls    import reverse
+
+from golem_messages import message
+
+from ..models       import UploadReport
+from ..models       import UploadRequest
+from ..models       import VerificationRequest
+from ..tasks        import verification_request_task
+
+
+class ConductorVerificationIntegrationTest(TestCase):
```

Comment:
there is some duplication in the tests... could you please extract it to fields? I mean for example: `self.file_path`, `self.verification_request`, `self.upload_request`, `self.compute_task_def`, etc.

Reply:
sorry, I am not sure what you mean by: 

> other objects which you have listed cannot because some tests depends on their existence.

could you please explain?

Reply:
For example, I can't create `upload_request` in setup because there are tests which rely on if it is not created.